nuget.config

```diff
@@ -4,6 +4,7 @@
     <clear />
     <add key="NuGet" value="https://api.nuget.org/v3/index.json" />
     <add key="ProjectReunion-Dependencies" value="https://pkgs.dev.azure.com/ms/ProjectReunion/_packaging/ProjectReunion-Dependencies/nuget/v3/index.json" />
+    <add key="dotnet-eng" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json" />
```

Comment:
><add key="dotnet-eng" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json" /> [](http://example.com/codeflow?start=4&length=115)

we are not supposed to add multiple sources

Reply:
Hmm, in that case I'll probably have to push the Helix SDK to the ProjectReunion-Dependencies feed.

Reply:
yeah, it appears what you are trying to add is internal feed, and we cannot use internal feed.

Reply:
Well the `dotnet-eng` feed is a public feed from the .NET Core team.
You can browse the feed here (without signing in):
https://dev.azure.com/dnceng/public/_packaging?_a=feed&feed=dotnet-eng

It's also publicly documented here:
https://github.com/dotnet/arcade/blob/main/src/Microsoft.DotNet.Helix/Sdk/Readme.md

But I think we can only have one source though, so might need to publish it to `ProjectReunion-Dependencies` feed anyways.

Reply:
oh, i thought it was for "Microsoft.Internal.WinUI.Helix", and assumed it's internal based on the name. 
Which feed does "Microsoft.Internal.WinUI.Helix" come from?

Reply:
I put it in the PR description :)
> This change uses the "Microsoft.Internal.WinUI.Helix" Nuget package, and removes the old/out-of-date checked-in copies of the Helix scripts from the repo. I've pushed this Nuget package to the "ProjectReunion-Dependencies" Nuget feed, so that other feeder repos (including this one) can use it.

@kmahone -- I think Hui has a good point that the word "internal" in the name sort of implies that it might not be public.
FWIW, the public WinUI repo is already using this Nuget package though. 
See `packages.config` here: https://github.com/microsoft/microsoft-ui-xaml/blob/main/build/Helix/packages.config#L8

Perhaps we could consider renaming it in the future maybe?

Reply:
Update: I pushed the "Microsoft.DotNet.Helix.Sdk" Nuget package to the "ProjectReunion-Dependencies" feed, so we don't need to add another feed to the `nuget.config`.

Reply:
It appears you just changed the version of "Microsoft.DotNet.Helix.Sdk". why was the need for a new feed (or adding it to a feed)?

Reply:
Right, I did bump the version for the "Microsoft.DotNet.Helix.Sdk". (I changed it to match the version used by the WinUI repo.)

The need for the new feed (or adding it to a feed) is coming from this step: "`NuGet restore Helix/RunTestsInHelix.proj`", in the file `build\AzurePipelinesTemplates\ProjectReunion-RunHelixTests-Job.yml`.

This does a Nuget restore on the .proj file, which pulls in the "Microsoft.DotNet.Helix.Sdk" package.
Before there was no Nuget restore being done -- so it wasn't being pulled in.

Reply:
Actually, from testing, it doesn't seem like doing the Nuget restore for the proj is *strictly necessary*.
Though I'm somewhat inclined to keep it, so that we're consistent with how it's being used in the PR-Internal repo and the WinUI repo.